core/app/models/spree/address.rb

```diff
@@ -18,19 +18,41 @@ class Address < Spree::Base
     alias_attribute :first_name, :firstname
     alias_attribute :last_name, :lastname
 
+    UNCOPIED_ATTRS = ['id', 'updated_at']
```

Comment:
At one point I had the idea that updated-via-copy addresses could share a created_at, but on second thought it should be in here. Your eagle eyes are appreciated.